src/events/crawler/scrapers/USA/KS/index.js

```diff
@@ -0,0 +1,187 @@
+    'Cloud County',
+    'Coffey County',
+    'Comanche County',
+    'Cowley County',
+    'Crawford County',
+    'Decatur County',
+    'Dickinson County',
+    'Doniphan County',
+    'Douglas County',
+    'Edwards County',
+    'Elk County',
+    'Ellis County',
+    'Ellsworth County',
+    'Finney County',
+    'Ford County',
+    'Franklin County',
+    'Geary County',
+    'Gove County',
+    'Graham County',
+    'Grant County',
+    'Gray County',
+    'Greeley County',
+    'Greenwood County',
+    'Hamilton County',
+    'Harper County',
+    'Harvey County',
+    'Haskell County',
+    'Hodgeman County',
+    'Jackson County',
+    'Jefferson County',
+    'Jewell County',
+    'Johnson County',
+    'Kearny County',
+    'Kingman County',
+    'Kiowa County',
+    'Labette County',
+    'Lane County',
+    'Leavenworth County',
+    'Lincoln County',
+    'Linn County',
+    'Logan County',
+    'Lyon County',
+    'Marion County',
+    'Marshall County',
+    'McPherson County',
+    'Meade County',
+    'Miami County',
+    'Mitchell County',
+    'Montgomery County',
+    'Morris County',
+    'Morton County',
+    'Nemaha County',
+    'Neosho County',
+    'Ness County',
+    'Norton County',
+    'Osage County',
+    'Osborne County',
+    'Ottawa County',
+    'Pawnee County',
+    'Phillips County',
+    'Pottawatomie County',
+    'Pratt County',
+    'Rawlins County',
+    'Reno County',
+    'Republic County',
+    'Rice County',
+    'Riley County',
+    'Rooks County',
+    'Rush County',
+    'Russell County',
+    'Saline County',
+    'Scott County',
+    'Sedgwick County',
+    'Seward County',
+    'Shawnee County',
+    'Sheridan County',
+    'Sherman County',
+    'Smith County',
+    'Stafford County',
+    'Stanton County',
+    'Stevens County',
+    'Sumner County',
+    'Thomas County',
+    'Trego County',
+    'Wabaunsee County',
+    'Wallace County',
+    'Washington County',
+    'Wichita County',
+    'Wilson County',
+    'Woodson County',
+    'Wyandotte County'
+  ],
+  scraper: {
+    '2020-03-18': async function() {
+      const date = process.env.SCRAPE_DATE || datetime.getYYYYMMDD();
+      const datePart = datetime.getMonthDYYYY(date);
+      this.url = `${this._baseUrl}COVID-19_${datePart}_.pdf`;
+
+      const body = await fetch.pdf(this.url, date, { alwaysRun: true });
+
+      if (body === null) {
+        // throw new Error(`No data for ${date}`);
+        return geography.addEmptyRegions([], this._counties, 'county');
+      }
+
+      const rows = pdfUtils.asRows(body).map(row => row.map(col => col.text));
+      // console.error(rows);
+
+      const counties = [];
+      const startIndex = rows.findIndex(cols => cols[0] && cols[0].includes('Positive Case Information')) + 2;
+      // console.error(startIndex);
+
+      for (let i = startIndex; i < rows.length; i++) {
+        const data = rows[i];
+        if (data[0].includes('County') || data[1] === parse.number(data[1])) {
```

Comment:
Careful. This will match any string that has the word County in it.
```suggestion
        if (data[0] === 'County' || data[1] === parse.number(data[1])) {
```

Reply:
Thanks.  That was on purpose.  I'm just trying to filter those rows that have something with "County" and/or those that have a number in the second column.  It's a bit clunky because they're being inconsistent about how they name/label the counties.  So, I think this should stay as is for now.